# Array of person positions
std_msgs/Header header
Box[] boxes
sensor_msgs/Image image